modules/heat_conduction/src/kernels/AnisoHeatConduction.C

```diff
@@ -8,103 +8,43 @@
 //* https://www.gnu.org/licenses/lgpl-2.1.html
 
 #include "AnisoHeatConduction.h"
-#include "MooseMesh.h"
 
 registerMooseObject("HeatConductionApp", AnisoHeatConduction);
 
+defineLegacyParams(AnisoHeatConduction);
+
 InputParameters
 AnisoHeatConduction::validParams()
 {
-  InputParameters params = Kernel::validParams();
-  params.set<bool>("use_displaced_mesh") = true;
-  return params;
+  InputParameters p = DerivativeMaterialInterface<Kernel>::validParams();
```

Comment:
we call this `params` everywhere else in the code. I suggest keeping with that convention

Reply:
Done